CONTRIBUTING.md

```diff
@@ -1,63 +1,63 @@
 # Contributing
 
-Thank you so much for contributing to 1Password. We appreciate your time and help. Here are some guidelines to help you get started.
+Thanks for your interest in contributing to the 1Password Connect Terraform Provider project! 🙌 We appreciate your time and effort. Here are some guidelines to help you get started.
 
 ## Building
 
-To build the 1Password Connect Terraform provider run the following:
+Run the following command to build the 1Password Connect Terraform Provider:
 
 ```sh
-$ go build .
+go build .
 ```
 
 This will create the `terraform-provider-onepassword` binary.
 
 ## Testing the Provider
 
-To run the go tests and check test coverage run the following:
+To run the Go tests and check test coverage run the following:
 
 ```sh
-$ go test -v ./... -cover
+go test -v ./... -cover
 ```
 
 ## Generating Documentation
 
-Documentation is generated for the provider using the [terraform documentation plugin](https://github.com/hashicorp/terraform-plugin-docs). This plugin uses the schema `Description` field in conjunction with the contents of the `/templates` and `/examples` folders to generate the `/docs` content.
+Documentation is generated for the provider using [terraform-plugin-docs](https://github.com/hashicorp/terraform-plugin-docs). This plugin uses the schema `Description` field in conjunction with the contents of the `/templates` and `/examples` folders to generate the `/docs` content.
 
-To regenerate the `/docs` markdown run:
+To regenerate the `/docs` Markdown run:
 
 ```sh
-$ go generate
+go generate
 ```
 
-## Installing Locally
+## Installing plugin locally
 
-To install the binary it must be copied to the appropriate terraform plugin directory. You may need to create the appropriate directories. For more information [see these Terraform 0.13 plugin docs](https://www.hashicorp.com/blog/automatic-installation-of-third-party-providers-with-terraform-0-13)
+To install the binary it must be copied to the appropriate Terraform plugin directory (note that you may need to create these directories first). You can refer to the [Terraform 0.13 plugin docs](https://www.hashicorp.com/blog/automatic-installation-of-third-party-providers-with-terraform-0-13) for more information.
 
-The current version of the provider is considered to be 0.2 and `darwin_amd64` should match your machines operating system and architecture in the format `$OS_$ARCH`. For example macOS is `darwin_amd64` and linux is `linux_amd64`.
+The current version of the provider is 1.2.0. Your machine's operating system and architecture should be specified in the `$OS_$ARCH` format. For example, macOS is `darwin_amd64` and Linux is `linux_amd64`.
```

Comment:
Question for reviewers: is it worth us updating the macOS example to account for Apple Silicon CPUs here? Below are example commands that I ran in my terminal (syntax: `command -flag = output`), but TBH I'm not sure which architecture we would reference here.

```sh
arch = arm64
uname -m = arm64
uname -p = arm
machine = arm64e
```

Reply:
Taking this even further, it's also possible to have Linux `arm64`, so if we want to update this, we should update this for all OSs. 
This makes me rethink whether we even have to provide these details. I think this was back when Go and Terraform worked best on Linux and macOS. Now I believe it works nicely on all platforms. Looking at the latest docs ([Locally install provider and verify with Terraform](https://developer.hashicorp.com/terraform/tutorials/providers-plugin-framework/providers-plugin-framework-provider#locally-install-provider-and-verify-with-terraform)) it seems that this process may be simplified and smoother, without the user needing to know the OS and arch in order to make it work.

Reply:
I'd agree with @edif2008 to not mention `arch`, as it should work on any platform

Reply:
Removed references to `arch` in the [2745e1b](https://github.com/1Password/terraform-provider-onepassword/pull/98/commits/2745e1bb6d8e1f29f4539b7453053d3aba064c52) commit.